### PR TITLE
fix:onOpenChange handler not being called in RangePicker for preset r…

### PR DESCRIPTION
**Pull request by leijingdao**
…anges(#11631)

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for branch `feature`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#11631: onOpenChange handler not being called in RangePicker for preset ranges (set via ranges prop to RangePicker)](https://issuehunt.io/repos/34526884/issues/11631)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

**Comment by afc163** [comment]
Please fix lint, use `npm run lint` to check your code style.

**Comment by leijingdao** [comment]
英文有点差，可以在这里直接用中文对话吗？我有一个疑问，想请教一下。 @afc163

**Comment by afc163** [comment]
Rebase 一下 master 以修复 ci。

File: components/date-picker/RangePicker.tsx [hunk 1]
```diff
@@ -184,10 +184,14 @@ class RangePicker extends React.Component<any, RangePickerState> {
 
     this.setValue(value, true);
 
-    const { onOk } = this.props;
+    const { onOk,onOpenChange } = this.props;
```
Comment (afc163) [inline components/date-picker/RangePicker.tsx]: 逗号后面应该有空格。

**Pull request merged**